test/release/examples/primers/chplvis/prog1.chpl

```diff
@@ -0,0 +1,10 @@
+//  Example 1 using visual debug
```

Comment:
Since this file will most likely be looked at first, I added comments to point to the .html file and to explain what the code was doing.